sorters/sortercontainer.h

```diff
@@ -30,9 +30,15 @@ class SorterContainer {
     virtual void onSorterRemoved(Sorter* sorter) = 0;
     virtual void onSortersCleared() = 0;
 
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
```

Comment:
Again the same check; I'd suggest just using `qsizetype` everywhere

Reply:
Same as above, it's not about `qsizetype` itself but qml list properties.